x-pack/plugins/actions/server/actions_config.ts

```diff
@@ -56,17 +56,17 @@ function disabledActionTypeErrorMessage(actionType: string) {
   });
 }
 
-function isAllowed({ allowedHosts }: ActionsConfigType, hostname: string): boolean {
+function isAllowed({ allowedHosts }: ActionsConfigType, hostname: string | null): boolean {
   const allowed = new Set(allowedHosts);
   if (allowed.has(AllowedHosts.Any)) return true;
-  if (allowed.has(hostname)) return true;
+  if (hostname && allowed.has(hostname)) return true;
```

Comment:
is the `hostname` check needed here?  Maybe for TS, but `.has()` supports `null` and `undefined` the way you want - I think you can even have `null` and `undefined` as values:

```console
> s = new Set()
Set(0) {}
> s.has(null)
false
> s.has(undefined)
false
```

Though I wonder if somehow a customer could inject a `null` into that list ... :-)

Reply:
Updated to remove extraneous check.